### ISSUE TITLE
A11y_ML .NET Model Builder_ Environment_ Screen Reader: Narrator/NVDA are readings "Environment Options" as  "tab item 1of 3" even when there is only one environment option present on the screen 

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1117]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #Benchmark; #Visual Studio; #Screen Reader; #Narrator; #NVDA; #WCAG1.3.1;

### Environment Details:
Microsoft Visual Studio 2019 Preview 16.8.0 Version 6.0

### Screen Readers:
Narrator; NVDA

### Repro Steps:

1.Start Screen Reader
2.Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Text Classification" scenario and select the scenario
6. Verify how Narrator/NVDA are reading the environment tab items present

### Actual Result:
Narrator/NVDA are readings "Environment Options" as  "tab item 1of 3" even when there is only one environment tab present on the screen. Narrator/NVDA are reading as "Environment Options Local CPU Environment Tab item 1 of 3 selected"

### Expected Result:
If there is only one environment option screen readers should read as "tab item 1 of 1" instead of reading as "Tab item 1 of 3"

### User Impact:
Screen Reader users will get confused if screen readers reads incorrect tab items count

### Attachment for Reference
[1117_ML .NET Model Builder_ NVDA is reading incorrect count of tab items.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5606360/1117_ML.NET.Model.Builder_.NVDA.is.reading.incorrect.count.of.tab.items.pptx)

[1117_ML .NET Model Builder_ Narrator is reading incorrect count of tab items.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5606335/1117_ML.NET.Model.Builder_.Narrator.is.reading.incorrect.count.of.tab.items.pptx)

**Comment by beccamc** [comment]
Build to test here: https://devdiv.visualstudio.com/_apis/resources/Containers/7181026/VSSetup?itemPath=VSSetup%2FMicrosoft.ML.ModelBuilder.vsix.

Download and install the vsix on your Visual studio instance. This build can be used to test all the "Ready to review" bugs (should be all)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in the given build and the issue is fixed. Hence closing the bug
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0 [30917.4.main]
![Screenshot (49)](https://user-images.githubusercontent.com/51435580/108030657-99cfde00-7055-11eb-8a04-80096b8a5f1e.png)
![Screenshot (50)](https://user-images.githubusercontent.com/51435580/108030663-9c323800-7055-11eb-8f9a-cb9803d6d44f.png)

**Comment by poojapatil36** [comment]
Github tag: #Closed;
